src/Sulu/Bundle/MediaBundle/Resources/public/js/validation/types/mediaSelection.js

```diff
@@ -15,8 +15,10 @@ define([
     'use strict';
 
     var dataChangedHandler = function(data, $el) {
-        App.emit('sulu.preview.update', $el, data);
-        App.emit('sulu.content.changed');
+        if (!!data && !!$el) {
```

Comment:
It looks to me like the real issue is here: https://github.com/sulu-io/sulu/blob/develop/src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/main.js#L246